doc/scapy/conf.py

```diff
@@ -100,6 +100,9 @@
 # Enable codeauthor and sectionauthor directives
 show_authors = True
 
+# Mock python-can
+autodoc_mock_imports = ["can"]
```

Comment:
Good job for finding this out, I didn't know it was builtin.

Reply:
Thanks!